resources/js/frontend.js

```
@@ -5,43 +5,44 @@ function formatDateTime(dateTimeString) {
         hour: '2-digit',
         minute: '2-digit'
-    }
-    const formatedDateTime = new Intl.DateTimeFormat('en-Us', options).format(new Date(dateTimeString));
-    return formatedDateTime;
+    };
+    return new Intl.DateTimeFormat('en-US', options).format(new Date(dateTimeString));
 }
 
-function scrollTobottom() {
+function scrollToBottom() {
+    let mainChatInbox = $('.wsus__chat_area_body');
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'mainChatInbox' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)